### ISSUE TITLE
Postie only supports POP3 w/o SSL and STARTTLS

### ISSUE THREAD

**Issue by RCEurope1** [creativecommons/europe#10]
... and that is offered by sort of not a single email provider anymore these days. Anything we can do about that?

**Comment by mvhaen** [comment]
When I go to the plugin settings > Mailserver, I can select these for the protocol:

POP3
IMAP
POP3-SSL
IMAP-SSL

Below that is a setting to enable TLS. Is that not what you are looking for?

Both are also advertised on the plugin homepage, so if these are not working correctly we should contact the developers of the plugin.

**Comment by RCEurope1** [comment]
No, that dropdown only shows POP3 for me and next to it it says "IMAP/IMAP-SSL/POP3-SSL unavailable". When I select "Yes" for TLS and save the settings, a message appears in the header saying "Warning: The POP3 connector does not support TLS".

**Comment by mvhaen** [comment]
From the postie faq (http://postieplugin.com/faq/):

 

 <http://postieplugin.com/faq/mail-host-requires-ssl-postie-will-allow-select-pop3-ssl-imap-ssl/> My mail host requires SSL, but Postie will not allow me to select POP3-SSL or IMAP-SSL

If you see the message Warning! Postie requires that imap be enabled if you are using IMAP, IMAP-SSL or POP3-SSL. you can still use Postie, but only with the POP protocol.

In order to use POP-SSL or IMAP-SSL you must have php-imap installed on your server. Ask your hosting provider about this or see  <http://postieplugin.com/faq/imap-extension-required-postie/> this if you manage your own server.

Specifically Postie looks to see if the function imap_mime_header_decode() is available which is part of the PHP IMAP library  <http://php.net/manual/en/book.imap.php> http://php.net/manual/en/book.imap.php

**Comment by mvhaen** [comment]
So the php-imap module needs to be installed on the server. I checked on my test-install this morning, which is why I didn’t spot this.

**Comment by RCEurope1** [comment]
ok, so it would be sufficiently clear to ask Matt or Rob to "install the php-imap module on the server where the CC Europe site sitd on", correct? Just checking :)

**Comment by RCEurope1** [comment]
"sits" I mean

**Comment by mvhaen** [comment]
John,

Yep. They should know what needs to be done. 

Regards,
Michael

**Comment by mattl** [comment]
@RCEurope1 What email account do you intend to use with this? 

WordPress has native post-by-email now, which just requires each user get a WordPress.com account, and will likely handle things better.

**Comment by mattl** [comment]
http://jetpack.me/support/post-by-email/ -- this seems like it can replace Postie.

**Comment by mvhaen** [comment]
Matt,

The reason that I went with Postie instead of Jetpack's post by e-mail is because from the the original discussions with Gwen and John I understood they wanted 1 e-mail address for posting. post-by-email requires each user to log in to the site and enable the feature. I believe the same goes for wordpress' own solution.

I did test it extensively up until a certain point and if the former is no longer an issue, it's a perfectly viable alternative.

Regards,
Michael

**Comment by mattl** [comment]
If that's the case, my question to John still applies... what email account do you intend to use with this?

I don't want this to be a creativecommons.org email account. If you wish to make and maintain an account with a third-party for this, you could, but perhaps the WordPress post by email feature also works instead?

**Comment by RCEurope1** [comment]
@mattl: well, as you had made yourself quite clear about that, I've registered a freemail account at the German hoster web.de

**Comment by mattl** [comment]
php5-imap installed.

**Comment by RCEurope1** [comment]
when I run the mailcheck script manually, I get:

"imap_open failed: Can't open mailbox {pop3.web.de:995/service=pop3/ssl/tls/novalidate-cert}: invalid remote specification
Mail Connection Time Out
Common Reasons: Server Down, Network Issue, Port/Protocol MisMatch 
The Server said:"

(the server saying nothing). Is that on our side or on theirs? can you tell?
